docs/mcp/per-server-false-positives.md

```
@@ -0,0 +1,245 @@
+itself, lift a specific pattern for one server. Parts 2 and 3 do that.
+
+## 2. Give the server a stable identity: `--server-name`
+
+```sh
+pipelock mcp proxy --server-name code-assistant --config code-assistant-pipelock.yaml -- code-assistant mcp-server
+```
+
+`--server-name <name>` assigns the wrapped server a stable identity. Pipelock
+uses it to build a per-server suppression **target** of the form:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)